lib/utils/spec-manager.ts

```diff
@@ -137,6 +137,42 @@ export class SpecManager {
     return obj;
   }
 
+  getOperationScopes(operationPtr:string) {
+    let that = this;
+    function getSecurityDefinition(name:string, scope:string) {
```

Comment:
Transform this function to fat-arrow function and remove `let that = this` trick:

```js
const getSecurityDefinition = (name: string, scope: string) => {
  // just use `this` here
}
```